### PR TITLE
Reverting back to gRPC version 0.7.1.

### PR DESCRIPTION
**Pull request by sduskis**
We're having a problem with 0.8.0 which seems to not be closing the batchPool executor in AbstractBigtableConnection properly.

### CODE REVIEW

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java [hunk 1]
```diff
@@ -137,6 +138,15 @@ private Cluster getCluster(BigtableClusterAdminClient client, String clusterName
     GetClusterRequest request = GetClusterRequest.newBuilder().setName(clusterName).build();
     try {
       return client.getCluster(request);
+    } catch (UncheckedExecutionException e) {
+      if (e.getCause() != null && e.getCause() instanceof StatusRuntimeException) {
```
Comment (kevinsi4508) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: I think u can do without "e.getCause() != null". If the cause is null, instanceof will be false.

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java [hunk 2]
```diff
@@ -137,6 +138,15 @@ private Cluster getCluster(BigtableClusterAdminClient client, String clusterName
...
+        Status status = ((StatusRuntimeException) e.getCause()).getStatus();
+        if (status.getCode() == Status.NOT_FOUND.getCode()) {
+          return null;
+        }
+      }
+      e.printStackTrace();
```
Comment (kevinsi4508) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: remove?

**Comment by kevinsi4508** [comment]
LGTM

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java (hunk 2 above)
Comment (sduskis) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: I'm on with this code in the cluster api test.

**Pull request merged**